src/components/Main.js

```diff
@@ -15,7 +16,7 @@ class Main extends Component {
 					<Menu/>
 				</HashRouter>
 			</AdminPortal>
-			<Route exact path="/" render={() => <Header/>} />
+			<Route exact path="/" render={() => <Dashboard/>} />
```

Comment:
Instead of this follow the pattern below to use the `component` prop rather than `render`.